### DISCUSSION TITLE
Windows系统托盘

### DISCUSSION THREAD

**Discussion by reserveword** [SeaHOH/GotoX#186] [问答/咨询] [answered]
直接运行根目录下的start.py会导致没有系统托盘图标，要运行launcher下的start.py才有系统托盘图标，感觉这样的设计不太合理啊
能不能改成运行根目录的start.py也有托盘图标啊

**Comment by SeaHOH** [comment, marked as answer]
有两个 vbs 文件 Windows 可以使用。
